tourstream/fti-portal-styleguide
# Ets 1165 background color

- Updated Bootstrap
- npm audit -fix is done as much as it can be done. There is still something to do and needs patience until fixes are comming or hands on.
- fti-bootstrap is no longer a dependency of this repository, instead it is inside the repository again and is build along with all the other styles as it was. Things are now one code review simpler

Note: This repository is exporting a style.css and a style.min.css which in the end will be important for the portal. Maybe we can come up with a simpler process in the future.

If you like to test it on you local branch:

- fetch this branch
- run `npm run build`
- go to ./dist-master/css/ and copy style.css and style.min.css
- inside the portal go to ./packages/fti-distribution/Recources/Public/Stylesheets/fti-portal-styleguide/
- delete the old files and paste the new ones there
- start your portal and make sure all browser caches are clean
